sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/azure/cognitiveservices/language/textanalytics/_generated/_text_analytics_api.py

```diff
@@ -0,0 +1,72 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from azure.core import PipelineClient
+from msrest import Serializer, Deserializer
+
+from ._configuration import TextAnalyticsAPIConfiguration
+from .operations import TextAnalyticsAPIOperationsMixin
+from . import models
+
+
+class TextAnalyticsAPI(TextAnalyticsAPIOperationsMixin):
+    """The Text Analytics API is a suite of text analytics web services built with best-in-class Microsoft machine learning algorithms.
+    The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase and entity extraction as well as language detection.
+    No training data is needed to use this API; just bring your text data.
+    This API uses advanced natural language processing techniques to deliver best in class predictions.
+    Further documentation can be found in https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview
+    This API is currently available in:
+    * Australia East - australiaeast.api.cognitive.microsoft.com
+    * Brazil South - brazilsouth.api.cognitive.microsoft.com
+    * Canada Central - canadacentral.api.cognitive.microsoft.com
+    * Central India - centralindia.api.cognitive.microsoft.com
+    * Central US - centralus.api.cognitive.microsoft.com
+    * East Asia - eastasia.api.cognitive.microsoft.com
+    * East US - eastus.api.cognitive.microsoft.com
+    * East US 2 - eastus2.api.cognitive.microsoft.com
+    * France Central - francecentral.api.cognitive.microsoft.com
+    * Japan East - japaneast.api.cognitive.microsoft.com
+    * Japan West - japanwest.api.cognitive.microsoft.com
+    * Korea Central - koreacentral.api.cognitive.microsoft.com
+    * North Central US - northcentralus.api.cognitive.microsoft.com
+    * North Europe - northeurope.api.cognitive.microsoft.com
+    * South Africa North - southafricanorth.api.cognitive.microsoft.com
+    * South Central US - southcentralus.api.cognitive.microsoft.com
+    * Southeast Asia - southeastasia.api.cognitive.microsoft.com
+    * UK South - uksouth.api.cognitive.microsoft.com
+    * West Central US - westcentralus.api.cognitive.microsoft.com
+    * West Europe - westeurope.api.cognitive.microsoft.com
+    * West US - westus.api.cognitive.microsoft.com
+    * West US 2 - westus2.api.cognitive.microsoft.com
```

Comment:
Is there an automated way to keep this in sync with what the service team publishes?  It seems like this list could change.

Reply:
This came directly from the v3 swagger. Not sure about an automated way to keep it in sync, I still need to review the docstrings for accuracy.

Reply:
Ah, I keep forgetting about autogen.  We should think about how often we plan to re-generate, and if it's likely that this will get out of sync, work with the service team to provide a link to up to date information instead of listing it in the swagger.

Reply:
There is a whole pipeline that automatically creates (or at least can create a) new PRs against your language of choice whenever a new openapi/swagger change is made in the azure-rest-api-specs repository. I'm sure that we can hook in to that in interesting ways.